src/Pages/AdminTaskTable/components/AdminTaskFilters.tsx

```diff
@@ -0,0 +1,211 @@
+  ChipGroup,
+  Flex,
+  FlexItem,
+  InputGroup,
+  SelectVariant,
+  TextInput,
+} from '@patternfly/react-core';
+import DropdownSelect from '../../../components/DropdownSelect/DropdownSelect';
+import { FilterIcon, SearchIcon } from '@patternfly/react-icons';
+import { global_BackgroundColor_100, global_secondary_color_100 } from '@patternfly/react-tokens';
+import Hide from '../../../components/Hide/Hide';
+import useDebounce from '../../../Hooks/useDebounce';
+import { createUseStyles } from 'react-jss';
+import { AdminTaskFilterData } from '../../../services/AdminTasks/AdminTaskApi';
+
+interface Props {
+  isLoading?: boolean;
+  setFilterData: (filterData: AdminTaskFilterData) => void;
+  filterData: AdminTaskFilterData;
+}
+
+const useStyles = createUseStyles({
+  chipsContainer: {
+    backgroundColor: global_BackgroundColor_100.value,
+    paddingTop: '16px',
+  },
+  clearFilters: {
+    marginLeft: '16px',
+  },
+  searchInput: {
+    paddingRight: '35px',
+    marginRight: '-23px',
+  },
+  searchIcon: {
+    color: global_secondary_color_100.value,
+    position: 'relative',
+    top: '3px',
+    left: '-5px',
+    pointerEvents: 'none',
+  },
+});
+
+const statusValues = ['Running', 'Failed', 'Completed', 'Canceled', 'Pending'];
+const filters = ['Account ID', 'Org ID', 'Status'];
+export type AdminTaskFilters = 'Account ID' | 'Org ID' | 'Status';
+
+const AdminTaskFilters = ({ isLoading, setFilterData, filterData }: Props) => {
+  const classes = useStyles();
+  const [filterType, setFilterType] = useState<AdminTaskFilters>('Account ID');
+  const [accountId, setAccountId] = useState('');
+  const [orgId, setOrgId] = useState('');
+  const [selectedStatuses, setSelectedStatuses] = useState<string[]>([]);
+
+  const clearFilters = () => {
+    setAccountId('');
+    setOrgId('');
+    setSelectedStatuses([]);
+    setFilterData({ accountId: '', orgId: '', statuses: [] });
+  };
+
+  const deleteItem = (id: string, chips, setChips) => {
+    const copyOfChips = [...chips];
+    const filteredCopy = copyOfChips.filter((chip) => chip !== id);
+    setChips(filteredCopy);
+  };
+
+  useEffect(() => {
+    // If the filters get cleared at the top level, sense that and clear them here.
+    if (
+      filterData.accountId === '' &&
+      filterData.orgId === '' &&
+      filterData.statuses.length === 0 &&
+      (accountId !== '' || orgId !== '' || selectedStatuses.length !== 0)
+    ) {
+      clearFilters();
+    }
+  }, [filterData]);
+
+  const {
+    accountId: debouncedAccountId,
+    orgId: debouncedOrgId,
+    selectedStatuses: debouncedSelectedStatuses,
+  } = useDebounce({
+    accountId,
+    orgId,
+    selectedStatuses,
+  });
+
+  useEffect(() => {
+    setFilterData({
+      accountId: debouncedAccountId,
+      orgId: debouncedOrgId,
+      statuses: debouncedSelectedStatuses,
+    });
+  }, [debouncedAccountId, debouncedOrgId, debouncedSelectedStatuses]);
+
+  const getSelectionByType = (): ReactElement => {
```

Comment:
Oh.. lol. Can you make this a component that accepts a "filterType" (just capitalize and give it one prop). 
And also do that in "ContentListFilters", calling react components as functions in the render works, but can lead to inefficient renders.